README.md

```diff
@@ -148,13 +174,38 @@ you also need:
 The build will work on any platform, but the tests for IE will be
 skipped silently if you are not building on Windows.
 
-## Common Tasks
+## Common Tasks (Bazel)
+
+To build the bulk of the Selenium binaries from source, run the 
+following command from the root folder:
+
+```sh
+bazel build java/... javascript/...
+```
+
+To build the grid deployment jar, run this command:
+
+```sh
+bazel build grid
+```
+
+To run tests within a particular area of the project, use the "test" command, followed
+by the folder or target:
 
-For an express build of the binaries we release, run the following from
+```sh
+bazel test java/...
+```
+
+Bazel's "test" command will run *all* tests in the package, including integration tests. Expect
+the ```test java/...``` to launch browsers and consume a considerable amount of time and resources.
```

Comment:
`bazel test --test_size_filters=small,medium java/...` will run fewer tests, yet still give you a reasonable level of confidence that everything is working.